lib/tacos/dependent_slices.py

```diff
@@ -250,10 +251,12 @@ def lines_to_paths(lines: Iterable[str]) -> Generator[OSPath]:
 def main() -> int:
     import fileinput
 
+    config = OSPath(".config/tacos-gha/" + sys.argv.pop())
```

Comment:
this string constant has an undesirable wetness to it

let's instead handle this in `from_config`

```python
        from sys import argv
        args = argv[1:]
        path_filter = PathFilter.from_config(*args)
```

This will properly give a (ugly but clear) too-many/too-few arguments error as appropriate.